katdal/visdatav4.py

```diff
@@ -20,6 +20,7 @@
 
 import numpy as np
 import katpoint
+import dask.array as da
```

Comment:
Looks like you're making dask a requirement now. Presumably it should be added to setup.py, and there is at least one place (in lazy_indexer.py) where you have a TODO to lift out a function-scoped import to the module level once katdal depends on dask.

Reply:
It's still optional since the whole v4 class is optional. But I should probably take my head out of the sand at some stage :-)